src/main/java/frc/robot/subsystems/DriveSystem.java

```diff
@@ -4,14 +4,89 @@
 
 package frc.robot.subsystems;
 
+import com.kauailabs.navx.frc.AHRS;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.CAN;
+import edu.wpi.first.wpilibj.drive.MecanumDrive;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class DriveSystem extends SubsystemBase {
+public class DriveSystem extends SubsystemBase implements Sendable {
+
+  private MecanumDrive mecanumDrive;
+  private CANSparkMax frontLeft;
+  private CANSparkMax backLeft;
+  private CANSparkMax frontRight;
+  private CANSparkMax backRight;
+
+  private double speedMultiplier = 0.8;
+
+  private boolean fieldOriented = true;
+  private AHRS navx;
+
   /** Creates a new DriveSystem. */
-  public DriveSystem() {}
+  public DriveSystem() 
+  {
+    frontLeft = new CANSparkMax(1, MotorType.kBrushless);
+    backLeft = new CANSparkMax(2, MotorType.kBrushless);
+    frontRight = new CANSparkMax(3, MotorType.kBrushless);
+    backRight = new CANSparkMax(4, MotorType.kBrushless);
+
+    mecanumDrive = new MecanumDrive(frontLeft, backLeft, frontRight, backRight);
+    navx = new AHRS();
```

Comment:
We have a new gyro.  Hooray!  But yeah, Navx is dead.

Reply:
https://docs.wpilib.org/en/stable/docs/software/hardware-apis/sensors/gyros-software.html#adis16470